torch/nn/intrinsic/quantized/modules/conv_relu.py

```diff
@@ -39,6 +39,10 @@ def _get_name(self):
 
     @classmethod
     def from_float(cls, mod):
+        if type(mod) == torch.nn.intrinsic.qat.ConvBnReLU1d:
+            mod.weight, mod.bias = fuse_conv_bn_weights(
+                mod.weight, mod.bias, mod.bn.running_mean, mod.bn.running_var,
+                mod.bn.eps, mod.bn.weight, mod.bn.bias)
```

Comment:
thoughts about moving this to a separate PR to minimize risk?

Reply:
sure